app/connectors/businessTaxReliefs/OtherReliefsConnector.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors.businessTaxReliefs
+
+import connectors.ConnectorConfig
+import connectors.errors.OtherReliefsSubmissionException
+import models.Done
+import models.businessTaxReliefs.OtherReliefs
+import play.api.Logging
+import play.api.http.Status._
+import uk.gov.hmrc.http.client.HttpClientV2
+import uk.gov.hmrc.http.{HeaderCarrier, HttpResponse, StringContextOps}
+import uk.gov.hmrc.http.HttpReads.Implicits.readRaw
+import utils.PagerDutyHelper.PagerDutyKeys.{FIVEXX_RESPONSE, FOURXX_RESPONSE, UNEXPECTED_RESPONSE}
+
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.{ExecutionContext, Future}
+
+
+trait OtherReliefsConnector {
+  def submit(otherReliefs: OtherReliefs)(implicit hc: HeaderCarrier): Future[Done]
+}
+
+@Singleton
+class OtherReliefsConnectorImpl @Inject()(config: ConnectorConfig,
+                                          httpClient: HttpClientV2
+                                          )(implicit ec: ExecutionContext) extends OtherReliefsConnector with Logging {
+
+  def submit(otherReliefs: OtherReliefs)(implicit hc: HeaderCarrier): Future[Done] = {
+    val url = s"${config.vcSessionServiceBaseUrl}/income-tax-session-data"
+    httpClient
+      .get(url"$url")
+      .execute[HttpResponse]
+      .map { res =>
+        res.status match {
+          case OK | ACCEPTED | NO_CONTENT =>
+            Done
+
+          case status if BAD_REQUEST >= status && status < INTERNAL_SERVER_ERROR =>
```

Comment:
Add:
`import uk.gov.hmrc.http.HttpErrorFunctions.is4xx`

```suggestion
          case status if is4xx(status) =>
```